packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -790,18 +792,25 @@ class HotRunner extends ResidentRunner {
       for (Uri uri in device.observatoryUris)
         printStatus('An Observatory debugger and profiler on $dname is available at: $uri');
     }
+    final String quitMessage = _didAttach
+        ? 'To detach, press "d" or to quit, press "q".'
```

Comment:
perhaps: `To detach, press "d"; to quit, press "q".`

Reply:
Done!